Add trilo xtest to users [SKIP_CI] (#12203)
mesonbuild/meson: docs/markdown/Users.md

 - [Terminology](https://github.com/billiob/terminology), a terminal emulator based on the Enlightenment Foundation Libraries
 - [Tilix](https://github.com/gnunn1/tilix), a tiling terminal emulator for Linux using GTK+ 3
 - [Tizonia](https://github.com/tizonia/tizonia-openmax-il), a command-line cloud music player for Linux with support for Spotify, Google Play Music, YouTube, SoundCloud, TuneIn, Plex servers and Chromecast devices

>  - [Trilobite XUnit Test](https://github.com/trilobite-stdlib/trilo-xtest-c), That test framework you wish you had when you started learning C. Has unit testing, UI testing, UX testing, AI testing, benchmarks, mocking, command-line interface, and much more.

Reviewer: ```suggestion
 - [Trilobite XUnit Test](https://github.com/trilobite-stdlib/trilo-xtest-c), a framework for testing C code
```
